src/main/java/org/cascadebot/cascadebot/utils/ColorUtils.kt

```diff
@@ -0,0 +1,126 @@
+package org.cascadebot.cascadebot.utils

```

Comment:
afaik Kotlin files should be under src/main/kotlin

Reply:
Not necessarily, they can be in src/main/java